src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -405,6 +405,48 @@ async Task IDurableOrchestrationClient.TerminateAsync(string instanceId, string
             }
         }
 
+        async Task IDurableOrchestrationClient.SuspendAsync(string instanceId, string reason)
```

Comment:
Is this the final iteration? I believe we removed the `IDurableOrchestrationClient.SuspendAsync` and `IDurableOrchestrationClient.ResumeAsync` from the DTFx layer, so I wasn't expecting to see these interface implementations here.

Reply:
@Julio-Arroyo

Reply:
Noting here what we talked about in-person (so that other interested parties are aware):

These suspend-resume APIs should remain here, as the interface `IDurableOrchestrationClient` is defined in the extension and not in DTFx.

Reply:
Re-requesting another review then :-)

Reply:
Yep, my bad :)